src/swatiShinde30/assignment3/Question4.java

```diff
@@ -0,0 +1,31 @@
+package swatiShinde30.assignment3;
+/*
+ *  WAP to find sum of user given number
+ */
+import java.util.Scanner;
+
+public class Question4 
+{
+	static int total;
+
+	static void totalAdd(String number)
+	{
+
+		for (int i = 0; i < number.length(); i++)
+		{
+			Character u;
```

Comment:
Here you can write char simple instead of Character class

Reply:
Make it simple like you get number in string format, 
1) first you convert it into Number
2) the take %10 to get last single number.
3) then you will add that number